aten/src/ATen/native/NonSymbolicBC.h

```diff
@@ -22,4 +22,5 @@ TORCH_API at::Tensor _embedding_bag_sparse_backward(const at::Tensor & grad, con
 TORCH_API at::Tensor value_selecting_reduction_backward(const at::Tensor & grad, int64_t dim, const at::Tensor & indices, at::IntArrayRef sizes, bool keepdim);
 TORCH_API at::Tensor trace_backward(const at::Tensor & grad, at::IntArrayRef sizes);
 TORCH_API at::Tensor index_select_backward(const at::Tensor & grad, at::IntArrayRef self_sizes, int64_t dim, const at::Tensor & index);
+TORCH_API std::vector<Tensor> tensor_split(const Tensor& self, IntArrayRef indices, int64_t dim);
```

Comment:
This has a small risk of internal breakage, right?

Although I just searched internal for `at::native::tensor_split`, and the only place that shows up is in static runtime (which is part of the OSS codebase). So.. I think this should be fine

Reply:
Ok, let's take the risk on this one. Thanks for checking!